arcane/src/arcane/accelerator/IndexSelecter.h

```diff
@@ -0,0 +1,136 @@
+#ifndef ACCELERATOR_INDEX_SELECTER_H
```

Comment:
- Add correct header.
- This file should not be here because it uses Arcane Internal API and so it can not be used outside of Arcane. A better place should be in materials/internal.
- Do not use `IParallelMng` in this class. Instead pass a RunQueue.
- Use only needed header to compile this file (for example no need to include `arcane/SubDomain.h`)
- Use full name for class member name (for example replace `m_is_acc_avl` by `m_is_accelerator_policy`)